redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -163,13 +163,13 @@ object Output {
 
   case object StringOutput extends Output[String] {
     protected def tryDecode(text: String): String =
-      if (text.startsWith("+"))
+      if (text.startsWith("$"))
```

Comment:
What happened here :confused: ?

Reply:
it just did not work this way ;) Redis returns $1 or $2 here

Reply:
The thing is, this is simple string output, it should start with `+`. We have `MultiStringOutput` that already works with `$` afaik.

Reply:
well I did this change based on redis results, what can I say :) otherwise zScore was not working, afaik this StringOutput is not used in other places

Reply:
Oh, you are right. Since this is the case, I propose to use `MultiStringOutput` here, and the do the "rename & nuke" in the follow-up pull request.